packages/twenty-server/src/modules/messaging/message-import-manager/services/messaging-partial-message-list-fetch.service.ts

```diff
@@ -54,7 +54,6 @@ export class MessagingPartialMessageListFetchService {
         },
         {
           throttleFailureCount: 0,
-          syncStageStartedAt: null,
```

Comment:
I'm not sure about this one, are you afraid of race conditions?

Reply:
actually I agree with this one as this is the role of the message-channel-sync status